docs/automated-tests/test-execution/quick-run-visual-debugger.md

```diff
@@ -1,24 +1,28 @@
 ---
 title: Enable Visual Debugger in Quick Execution
 page_title: Enable Visual Debugger in Quick Execution
-description: "Test Studio Quick test Execution. Test Studio test run. Visual debugger options in Test Studio for quick run. Debug the test during its execution."
+description: "Learn how to enable the Visual Debugger during quick test execution and how to debug a test during its execution."
 position: 5
 ---
 # Enable Visual Debugger
 
-The **Test Ribbon** in Test Studio allows you to trigger a quick test execution. Along with that it provides quick access to some useful project settings.
+The **Test** ribbon in Test Studio allows you to trigger a quick test execution and to adjust the debugging options for your next test run.
 
-The __Visual Debugger__ is a useful tool to help you explore the application, while a test is being executed. It provides different options when to pause the test run and allows you to explore the current DOM tree, so far generated results, etc. The debugging options for your next test run can be controlled from the **Test** ribbon in Test Studio.
+The __Visual Debugger__ helps you to explore the application while a test is executing. It allows you to:
+
+* Pause the test run.
+
+* Explore the current DOM tree.
+
+* Review the generated results up to the current moment.
 
 ## Visual Debugger
```

Comment:
One heading is plenty of space. Don't use multiple headings with a sparse amount of text.